plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -8564,6 +8564,35 @@ public void testCreateTableWithDataLocationButObjectStoreDisabled()
                 "Data location can only be set when object store is enabled");
     }
 
+    public void testReplaceTableWithMissingMetadataFile()
```

Comment:
Could you update `testCorruptedTableLocation` instead? Also, it would be nice to test `CREATE OR REPLACE TABLE AS SELECT` as well.